src/security/WSSecurityCert.ts

```diff
@@ -111,15 +112,19 @@ export class WSSecurityCert implements ISecurity {
       key: privatePEM,
       passphrase: password,
     };
+    this.keyInfoId = `KI-${generateId()}`;
```

Comment:
I don't think that `id` in `KeyInfo` is just a random string. My understanding from the spec is that this is an id that references element from `SignedInfo`.

Reply:
removed `keyInfoId` creation. letting `xml-crypto` handle it.